src/experimentEnrollment.js

```diff
@@ -4,20 +4,21 @@ export default {
 
   enrollInExperiment(component, child, variationName) {
     if (variationName && child.props.name === variationName) {
-      component.exposedVariation = child;
+      component.selectedVariation = child;
     } else if (child.props.displayName === DEFAULT_EXPERIMENT_COMPONENT) {
       component.defaultComponent = child;
     }
     return component;
   },
-  
-  getExposedExperimentVariation(childrenComponents, variationName) {
+
+  getVariation(childrenComponents, variationName) {
+    // I don't understand what this line does.
     if (!childrenComponents.reduce) {
```

Comment:
Right, but as mentioned in the other comment I left the following component structure is possible and valid way of implementing the following experiment: Which location of a Call to Action (CTA) in the header of the homepage leads to a higher conversion rate given that one already exists in the search bar? Suppose we want to run a simple A/B test between the existing CTA in the search bar and one in the nav but they are each their own separate React components. We would ideally want to be able to write the code as follows:

```javascript
var Header = React.createClass({
  render() {
    return {
      (
        <div>
          <Nav />
          <SearchBar />
        </div>
      )
    };
  }
});

var SearchBar = React.createClass({
  render() {
    return {
      (
        ...
        <Experiment experimentClass={foo} param="bar">
          <Variation name="foo">
            foo
          </Variation>
        </Experiment>
      )
    }
  }
});

var Nav = React.createClass({
  render() {
    return {
      (
        ...
        <Experiment experimentClass={foo} param="bar">
          <Variation name="bar">
            bar
          </Variation>
        </Experiment>
      )
    }
  }
});
```
 
We could conceivably just have blank components for each variation in both of the Experiment classes but it feels overly verbose and clunky when we're implementing more than a simple A/B experiment.